File: tests/wasm2lang_10_bulk_memory.harness.java
// Java test harness for wasm2lang_10_bulk_memory.
{
    java.util.Map<String, Object> foreign = new java.util.LinkedHashMap<>();

    WasmModule mod = new WasmModule(foreign, memBuffer);
    mod.alignHeapTop();
    mod.exerciseBulkMemory(mod.getHeapTop());

    w2lDumpCRC(memBuffer);
}

/exit

File: tests/wasm2lang_10_bulk_memory.harness.mjs
'use strict';

const moduleImports = {};

const runTest = function (buff, out, exports) {
  exports.alignHeapTop();
  exports.exerciseBulkMemory(exports.getHeapTop());
};

const dumpMemory = true;

export { dumpMemory, moduleImports, runTest };
